packages/cli/commands/projects.js

```diff
@@ -4,16 +4,21 @@ const {
   addOverwriteOptions,
 } = require('../lib/commonOpts');
 const deploy = require('./projects/deploy');
+const init = require('./projects/init');
+const upload = require('./projects/upload');
 
-exports.command = 'projects';
+exports.command = 'project';
 exports.describe = false; //'Commands for working with projects';
 
 exports.builder = yargs => {
   addOverwriteOptions(yargs, true);
   addConfigOptions(yargs, true);
   addAccountOptions(yargs, true);
 
+  // TODO: deploy must be updated
   yargs.command(deploy).demandCommand(1, '');
```

Comment:
Not currently used in the flow. Will need to be adjusted eventually, but will mostly stay the same so keeping this for now.